docs/cli.rst

```diff
@@ -1,6 +1,6 @@
 CLI Reference
 =============
 
-.. click:: howfairis.cli:cli
+.. click:: howfairis.cli.cli:cli
```

Comment:
I like this better than `module_cli.cli` like I did in `233-doc-repo-fixes`

Reply:
not only better, without it the cli docs are broken.

Reply:
done